types/fs-extra/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "dtslint/dt.json"
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "unified-signatures": false
```

Comment:
as this project is related to `@types/node` I believe it's better for both to keep a similar shape rather than to optimize for unified-signatures

Reply:
Yes, that's correct, but it is also better to use inline exclusion, not general rule, so per-readme for DT:
https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson
@types/nodes are just so complex, it would be unrealistic to exclude via inline comments,
also we culd just do something like (without modifying tslint):
```ts
export namespace realpath {
    const native: {
        (path: PathLike, options?: fs.BaseEncodingOptions | BufferEncoding | null): Promise<string>;
        (path: PathLike, options: fs.BaseEncodingOptions | string | undefined | null): Promise<string | Buffer>;
        (path: PathLike, options: fs.BufferEncodingOption): Promise<Buffer>;
    } & typeof fs.realpath.native;
```
(the expected return types would be different and there is no inferred return type). There would be no need to retype `fs` version (dry)

Reply:
I think this feedback makes sense, punting on merging to give discussion a bit more time

Reply:
Adjusted as proposed 👍